README.md

```diff
@@ -21,7 +21,7 @@ $ npm install nft-did-resolver
 ### Usage
 
 ```typescript
-import NftResolver, { NftResolverConfig } from 'nft-did-resolver'
+import { getResolver, NftResolverConfig } from 'nft-did-resolver'
```

Comment:
Is there no default export anymore? If so we should bump major version of the package.